libs/designer/src/lib/ui/templates/connections/displayConnections.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from '@fluentui/react-components';
+import type { Template } from '@microsoft/logic-apps-shared';
+
+export interface DisplayConnectionsProps {
+  connections: Record<string, Template.Connection>;
+}
+
+export const DisplayConnections = ({ connections }: DisplayConnectionsProps) => {
+  return (
+    <div>
```

Comment:
Nit: maybe use empty tags?

Reply:
This component is just a placeholder for displaying connections and will have to be changed again, but I can take those NIT in account to keep it clean!